tests/del_test.go

```diff
@@ -0,0 +1,45 @@
+package tests
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestDel(t *testing.T) {
+	conn := getLocalConnection()
+	defer conn.Close()
+
+	testCases := []struct {
+		name     string
+		commands []string
+		expected []interface{}
+	}{
+		{
+			name:     "DEL with set key",
+			commands: []string{"SET k1 v1", "DEL k1"},
+			expected: []interface{}{"OK", int64(1)},
+		},
+		{
+			name:     "DEL with multiple keys",
+			commands: []string{"SET k1 v1", "SET k2 v2", "DEL k1 k2"},
```

Comment:
Please also validate that the get call returns nil for deleted keys

Reply:
Resolved, @JyotinderSingh kindly re-review